tdx-tdcall/src/tdreport.rs

```diff
@@ -30,12 +30,12 @@ pub struct ReportType {
 #[derive(Debug, Pread, Pwrite)]
 pub struct ReportMac {
     pub report_type: ReportType,
-    reserved0: [u8; 12],
+    pub reserved0: [u8; 12],
```

Comment:
It would be better to implement Default or new() for ReportMac instead of export these reserved fields.

Reply:
Thank you! Since I need to use the struct to build a new struct, so I prefer to add 'pub' even it is not the best solution.

Reply:
You could also use "xx::new()" or `Default::default()` to create new data structs, so missed the point why you must have access to these fields.